packages/kit/src/exports/vite/dev/index.js

```diff
@@ -64,9 +70,13 @@ export async function dev(vite, vite_config, svelte_config) {
 	let manifest_error = null;
 
 	/** @param {string} url */
-	async function loud_ssr_load_module(url) {
+	async function load_ssr_load_module(url) {
+		if (!isRunnableDevEnvironment(viteEnv)) {
```

Comment:
repeated logic because these functions are all hoisted before we've done our type guard on L38